gamemode/modules/voting/sv_votes.lua

```diff
@@ -50,7 +50,8 @@ function Vote:handleEnd()
 end
 
 function Vote:getFilter()
-	local filter = RecipientFilter()
+	--local filter = RecipientFilter()
```

Comment:
It's there to show legacy functionality. Eventually, it should just be replaced/removed.